src/FreeDSx/Snmp/Message/EngineId.php

```diff
@@ -37,6 +37,7 @@ class EngineId
 
     public const FORMAT_OCTET = 5;
 
+    /** @var string */
```

Comment:
Could you format this docblock to be the same as the rest? ie. 

```php
    /**
     * @var string
     */
```

Reply:
Ayright, I'm used to that compact format when there's only type